### DISCUSSION TITLE
The display on idea is abnormal

### DISCUSSION THREAD

**Discussion by EZzer0** [starship/starship#5543] [Q&A]
idea:
<img width="503" alt="image" src="https://github.com/starship/starship/assets/47420680/7754f616-848a-422a-b8b8-2f777b059886">
vscode:
<img width="500" alt="image" src="https://github.com/starship/starship/assets/47420680/7216d0c7-6111-49df-8634-68268983ad5b">
powershell
<img width="593" alt="image" src="https://github.com/starship/starship/assets/47420680/6a8ae21c-7683-4ad8-82e5-502b1b1d6ee8">

starship.toml
format = """
[](fg:#7ce38b)\
[ E~z ](bg:#7ce38b fg:#121212)\
[](fg:#7ce38b bg:#21262d)\
$time\
[](fg:#21262d bg:#a2d2fb)\
$directory\
[](fg:#a2d2fb bg:#21262d)\
$git_branch\
$git_status\
[ ](fg:#21262d)\
"""
Please tell me how to solve this display problem

**Comment by davidkna** [comment]
Idea is known to have sub-optimal font rendering in its terminal. IIRC there was an option to use a different rendering engine which may or may not work better.
